tests/clients/test_dapr_grpc_client.py

```diff
@@ -681,6 +682,58 @@ def test_unlock_input_validation(self):
             with self.assertRaises(ValueError):
                 dapr.unlock(store_name, resource_id, invalid_input)
 
+    #
+    # Tests for workflow
+    #
+
+    def test_workflow(self):
+        dapr = DaprGrpcClient(f'localhost:{self.server_port}')
+        # Sane parameters
+        workflow_name = 'testWorkflow'
+        event_name = 'eventName'
+        instance_id = str(uuid.uuid4())
+        workflow_component = "dapr"
+        input = "paperclips"
+        event_data = "cars"
+        # Start the workflow
+        start_response = dapr.start_workflow(instance_id, workflow_name, workflow_component,
+                                             input.encode('utf-8'), None)
```

Comment:
Related to my other comment, I think we want to allow users to just pass `input` here.